registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -650,6 +653,7 @@
     "sha256:8fbf1ba64f76d8d697514898ec85ef46c47befc63e8f459157f2938969ba1c2a": ["v1.10.2"]
     "sha256:929d39dcc639efcd63572acff37137e591451f57c0338cf05605b3983a4a672c": ["v1.10.3"]
     "sha256:74ae645824a63b948db10ad70e98c3ce8ee268caa07003216ca04fcc88a0d8e2": ["v1.10.4"]
+    "sha256:315e7aebeb31832e577ef6303b08ee8ca0bb9377be82cc942352e2b049272dab": ["v1.11.0"]
```

Comment:
SHA `315e7aebeb31` matches kubeadm-control-plane-controller v1.11.0 at the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/kubeadm-control-plane-controller@sha256:315e7aebeb31832e577ef6303b08ee8ca0bb9377be82cc942352e2b049272dab/details?inv=1&invt=Ab56Hw).